Integrations/TinkoffIntegrationLib/TinkoffBankBroker.cs

```diff
@@ -68,5 +63,25 @@ public IEnumerable<Instrument> GetInstruments(DTO.MarketBrokerObjects.Instrument
 
             return instruments;
         }
+
+        public Transaction Trade(InternalTradeRequest request)
+        {
+            try
+            {
+                var transaction = request.Transaction;
+                var operation = transaction.Operation == DTO.MarketBrokerObjects.OperationType.Buy ? OperationType.Buy : OperationType.Sell;
+                var order = new LimitOrder(transaction.Figi, transaction.Count, operation, transaction.Price);
+                var result = context.PlaceLimitOrderAsync(order).Result;
+                transaction.DateTime = DateTime.Now;
+                transaction.IsSuccess = result.Status == OrderStatus.Fill ? true : false;
+                return transaction;
+            }
+            catch
+            {
+                throw new BadRequestException();
+            }
+
```

Comment:
How do you get these lines

Reply:
I don't know either :) 
Probably, had a lot of deletion

Reply:
fixed